etcdserver/etcdserverpb/rpc.pb.go

```diff
@@ -296,7 +296,7 @@ type RangeRequest struct {
 	// greater mod revisions will be filtered away.
 	MaxModRevision int64 `protobuf:"varint,11,opt,name=max_mod_revision,json=maxModRevision,proto3" json:"max_mod_revision,omitempty"`
 	// min_create_revision is the lower bound for returned key create revisions; all keys with
-	// lesser create trevisions will be filtered away.
```

Comment:
@Revolution1, welcome thank you for your PR. Just a quick question, how did you update this file?

Reply:
Since there is only one word has changed(of the doc), I just replaced the word in this file instead of regenerate it. Should i regenerate it ?

Reply:
I think it's fine honestly but more important that you understood the process required.

Reply:
thanks, I will remenber that 😄